src/groundStation/system.py

```diff
@@ -97,9 +97,9 @@ def __init__(self):
                         'inoutInfo': {
                             'args': None,
                             'returns': {
-                                'err': '>b',
-                                'timestampInS': '>u4',
-                                'secondFraction': '>u4'
+                                'err': '>b',  # error is 2 (wrong pps)
+                                #'timestampInS': '<u4',
+                                #'secondFraction': '<u4'
```

Comment:
Does the EPS not return these values?

Reply:
Yeah, Nick told me it can't really give out PPS time right now. So, for now, I commented it out. The current implementation of the ground station would fail if it asks for more data than there is in the packet. And EPS only returns the error when there is no timestamp to report. So, I had to comment this out to read the error code.

Reply:
Makes sense